CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ install(DIRECTORY cmake
   PATTERN "catkinConfig.cmake.in" EXCLUDE)
 
 catkin_python_setup()
+execute_process(COMMAND sleep 10)
```

Comment:
Oh, sorry , that's a debug code. removed .

--
◉ Kei Okada


On Sun, Feb 1, 2015 at 3:03 PM, Dirk Thomas <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/ros/catkin/pull/705#discussion_r23895172>:
>
> > @@ -21,6 +21,7 @@ install(DIRECTORY cmake
> >    PATTERN "catkinConfig.cmake.in" EXCLUDE)
> >
> >  catkin_python_setup()
> > +execute_process(COMMAND sleep 10)
>
> ???
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros/catkin/pull/705/files#r23895172>.
>